src/local/motion_test/test_BSpline_transformations.cxx

```diff
@@ -0,0 +1,185 @@
+/*
+    Copyright (C) 2018, University College London
+    This file is part of STIR.
+
+    This file is free software; you can redistribute it and/or modify
+    it under the terms of the GNU Lesser General Public License as published by
+    the Free Software Foundation; either version 2.1 of the License, or
+    (at your option) any later version.
+
+    This file is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Lesser General Public License for more details.
+
+    See STIR/LICENSE.txt for details
+*/
+/*!
+
+  \file
+  \ingroup stir::motion_test
+
+  \brief Test program for transformations with NonRigidObjectTransformationUsingBSplines
```

Comment:
needs `stir::` prefix, otherwise doxygen won't link. (The doc is not within the namespace)